wherehows-web/tests/integration/components/datasets/containers/dataset-health-test.js

```diff
@@ -1,13 +1,20 @@
-import { module, test } from 'qunit';
-import { setupRenderingTest } from 'ember-qunit';
-import { render } from '@ember/test-helpers';
+import { moduleForComponent, test } from 'ember-qunit';
```

Comment:
We've moved to the newer syntax, since `moduleForComponent` is deprecated. Also, `ember-native-dom-helpers` is deprecated, you can use `@ember/test-helpers` which provided a 1-1 api

Reply:
👍 , yea it's working now, was a pain to get working haha